content/participants/ttjc@sidrakshe28/index.md

```diff
@@ -0,0 +1,48 @@
+---
+title: Siddhika Rakshe
+type: "participant"
+name: "Siddhika Rakshe"
+portfolio: 
+github-link: "https://github.com/sidrakshe28"
+linkedin-link: " https://www.linkedin.com/in/siddhika28/"
+looking-for: "internship"
+description: " well-versed with HTML|CSS|UI-UX|Android app development|learning React"
+---
+
+Hi! I am a Siddhika Rakshe, let me walk you through my intro, projects and blogs.
```

Comment:
Hey! @sidrakshe28 a small mistake here:
* I am **a** Siddhika ...

Reply:
Sure .thanks for informing me.